back-end/java/main/java/src/Parsing/RoomParser.java

```diff
@@ -0,0 +1,153 @@
+package src.Parsing;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Iterator;
+
+import org.apache.commons.csv.CSVRecord;
+import src.DormRoom.*;
+
+public class RoomParser {
+
+    public Parser parser;
+
+    public RoomParser(String path) throws FileNotFoundException, IOException {
+        this.parser = new Parser(path);
+    }
+
+    public ArrayList<IDormRoom> getRooms(){
+        ArrayList<IDormRoom> result = new ArrayList<>();
+
+
```

Comment:
2 lines of space between this is not needed. Could just be one line. I know this is pedantic.